crates/bgp-pkt/src/capabilities.rs

```diff
@@ -338,6 +338,10 @@ impl AddPathCapability {
     pub const fn address_families(&self) -> &Vec<AddPathAddressFamily> {
         &self.address_families
     }
+
+    pub fn address_families_mut(&mut self) -> &mut Vec<AddPathAddressFamily> {
```

Comment:
what's this for? I tried to keep the packets immutable. Also unrelated to this PR